src/webgl/shaders/font.frag

```diff
@@ -0,0 +1,189 @@
+#extension GL_OES_standard_derivatives : enable
```

Comment:
i'll comment this once we have glsl minification in the build

Reply:
Yes, this file needs comments. 👍

Reply:
i added issue #3114

Reply:
... and PR #3116